cmd/osmosisd/cmd/root.go

```diff
@@ -863,7 +863,6 @@ func queryCommand(moduleBasics module.BasicManager) *cobra.Command {
 	)
 
 	// UNFORKING v2 TODO: Auto CLI claims we can remove this, but was having issues with AddTxCommands counterpart. See line for comment.
```

Comment:
Can we remove this UNFORKING v2 TODO as well then?

Reply:
Ah haha was just doing so! Thank you!